packages/dev/core/src/Misc/uvprojector.ts

```diff
@@ -0,0 +1,136 @@
+import { VertexBuffer } from "../Buffers";
+import { Ray } from "../Culling/ray";
+import { Vector3 } from "../Maths";
+import { AbstractMesh } from "../Meshes/abstractMesh";
+import { CreateBox } from "../Meshes/Builders/boxBuilder";
+import { Mesh } from "../Meshes/mesh";
+import { Scene } from "../scene";
+import { FloatArray, Nullable } from "../types";
+
+export class Uvprojector {
+    /**
+     * Project uv from a reference mesh
+     * @param target the mesh to project the uv to
+     * @param reference the reference mesh to project the uv from
+     * @param scalefact the bounding box scale for the reference mesh
+     * @param scene
+     */
+     public static ProjectUV(target: Mesh, reference: Nullable<Mesh>, scalefact: number, scene: Scene): void {

```

Comment:
I think we can directly export the function (right @RaananW ? or is it ok like that?)

Reply:
We should slowly stop using static members on classes and move to exporting from the module itself. We are still not doing it though, and it'll be globally done as party of the esm package work. So far now I'm fine with both ways of dealing with that. 
For the Babylon namespace, it's better to have it public static to not pollute the global namespace.